src/test/bench_test/benchmark.cpp

```diff
@@ -152,6 +155,57 @@ void benchmark::write_random(thread_arg *thread)
     thread->stats.add_bytes(bytes);
 }
 
+void benchmark::multi_set_random(thread_arg *thread)
+{
+    if (FLAGS_benchmark_num % FLAGS_multi_count != 0) {
+        fmt::print(stderr,
+                   "num {} should be a multiple of multi_count({}).\n",
+                   FLAGS_benchmark_num,
+                   FLAGS_multi_count);
+        dsn_exit(1);
+    }
+
+    // do write operation num times
```

Comment:
For comments, it's recommend to use upper case for the leading letter, and ended by a dot.